feature-utils/poly-look/src/react-components/contexts/poly-import.jsx

```diff
@@ -48,7 +48,7 @@ export const PolyImportProvider = ({
 
   const handleRemoveFile = (fileID) => {
     setAccount(null);
-    return storage.removeFile(fileID);
+    storage.removeFile(fileID);
```

Comment:
It appears that return value is actually being used, at least in _facebookImport_. I didn't check, but I'll just resume `storage.removeFile` is either  `async` or explicitly returns a promise. So this function explicitly returns said promise so that it can be `await`ed.

A possibly more clear/modern way of writing this would be:

```
const handelRemoveFile = async (fileID) => {
    setAccount(null);
    await storage.removeFile(fileID);
};
```